packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -291,7 +322,17 @@ abstract class WidgetsBinding extends BindingBase implements GestureBinding, Ren
       debugShortDescription: '[root]',
       child: app
     ).attachToRenderTree(buildOwner, renderViewElement);
+    assert(() {
+      if (debugPrintBeginFrameBanner)
+        debugPrint('━━━━━━━┫ Begin Preemptive Frame ┣━━━━━━━');
```

Comment:
Not sure what the term "preemptive" means in this context.  What frame are we preempting?